motor_info:
  min_pwm: 0.6 # 1200 [ms]
  max_pwm: 0.975 # 1950 [ms]
  min_thrust: 0 #  [N]
  force_landing_thrust: 1.1 #[N]
  m_f_rate: -0.011
  pwm_conversion_mode: 0 # SQRT Mode
  vel_ref_num: 4
  ref1:
    voltage: 11
    max_thrust: 6.2 # N
    polynominal2: 0.005302399 # 0.0005302399 # x10
    polynominal1: 0.763122768 # 0.0763122768 # x10
    polynominal0: -6.1211512186 # -6.1211512186 # x1

  ref2:
    voltage: 12
    max_thrust: 7.1 # N
    polynominal2:  0.004907324 # 0.0004907324 # x10
    polynominal1:  1.030117891 # 0.1030117891 # x10
    polynominal0: -7.5057784799 # -7.5057784799 # x1

  ref3:
    voltage: 14
    max_thrust: 8.5 # N
    polynominal2: 0.004142648 # 0.0004142648 # x10
    polynominal1: 1.452873062 # 0.1452873062 # x10
    polynominal0: -9.4010666512 # -9.4010666512 # x1

  ref4:
    voltage: 15
    max_thrust: 9.2 # N
    polynominal2: 0.007035072 # 0.0007035072 # x10
    polynominal1: 1.297139996 # 0.1297139996 # x10
    polynominal0: -9.5299513121 # -9.5299513121 # x1
